### ISSUE TITLE
"Google chrome 60+ required" error

### ISSUE THREAD

**Issue by R1ckj** [mimbrero/whatsapp-desktop-linux#41]
When opening the app I get the error message: "Whatsapp works with google chrome 60+" and there is a link to upgrade chrome. This error cannot be closed, so the app is not working for me anymore

**Comment by masemoel** [comment]
I had this error before and reinstalling the app fixed it, but now it doesn't work as well. This app needs an update - I hope it soon :(

**Comment by mimbrero** [comment]
Are you sure you are running the latest version? What packaging format?

**Comment by R1ckj** [comment]
Yes, I am running the latest version of the flatpak. But actually I tried to open the app again today and it was working..

**Comment by sly4** [comment]
Does it matter that web.whatsapp.com is down?  The QR is not scanning for me trying to fallback to whatsapp web.  I was getting the Chrome 60+ required also while running Version 110.0.5481.177 (Official Build) (64-bit).
https://www.isitdownrightnow.com/web.whatsapp.com.html

**Comment by oasido** [comment]
Problem still occurs on Pop!_OS 22.04 LTS x86_64

**Comment by sly4** [comment]
> Does it matter that web.whatsapp.com is down? The QR is not scanning for me trying to fallback to whatsapp web. I was getting the Chrome 60+ required also while running Version 110.0.5481.177 (Official Build) (64-bit). https://www.isitdownrightnow.com/web.whatsapp.com.html

FIXED: My problem was that I had DarkReader extension enabled.  After disabling for web.whatsapp.com, the QR scan worked correctly.
